src/test/java/de/tum/in/test/integration/testuser/ITASTTestUser.java

```diff
@@ -0,0 +1,487 @@
+package de.tum.in.test.integration.testuser;
+
+import org.junit.jupiter.api.Test;
+
+import de.tum.in.test.api.AddTrustedPackage;
+import de.tum.in.test.api.StrictTimeout;
+import de.tum.in.test.api.WhitelistPath;
+import de.tum.in.test.api.ast.asserting.UnwantedNodesAssert;
+import de.tum.in.test.api.ast.tool.PathTool;
+import de.tum.in.test.api.ast.type.*;
+import de.tum.in.test.api.jupiter.Public;
+import de.tum.in.test.api.localization.UseLocale;
+
+//<editor-fold desc="at">
+@AddTrustedPackage("com.github.javaparser.**")
+@Public
+@UseLocale("en")
+@StrictTimeout(10)
+@WhitelistPath("")
+//</editor-fold>
+public class ITASTTestUser {
+
+	// <editor-fold desc="For-Loop">
```

Comment:
If there are implicit groups, consider removing the markers and meaning the groups explicit: https://junit.org/junit5/docs/current/user-guide/#writing-tests-nested

Reply:
[Implemented part of the new feedback](https://github.com/ls1intum/Ares/pull/248/commits/ead5d75f0814dbd3738c57b058de8759b980e46d): Replaced editor-folds with nested classes.